site2/website-next/scripts/split-version-build.sh

```diff
@@ -76,5 +76,6 @@ done <scripts/.versions
 
 cp -r build-assets/* build/assets/
 rm -rf build-assets
+cp static/.htaccess build/
```

Comment:
Shall we copy all static files to build here and specially handle the hidden files like `.htaccess`?

In this way, we don't have to specially copy static files in:

https://github.com/apache/pulsar-site/blob/c30db65192c9f1791bf4bd2cf1b300124643933a/site2/tools/build-site.sh#L45-L62

Reply:
good idea

Reply:
It seems that we don't need this line then.